src/dynatrace/azext_dynatrace/aaz/latest/dynatrace/monitor/tag_rule/_update.py

```diff
@@ -143,7 +168,12 @@ def _build_args_filtering_tag_update(cls, _schema):
 
     def _execute_operations(self):
         self.pre_operations()
-        self.TagRulesUpdate(ctx=self.ctx)()
+        self.TagRulesGet(ctx=self.ctx)()
+        self.pre_instance_update(self.ctx.vars.instance)
+        self.InstanceUpdateByJson(ctx=self.ctx)()
+        self.InstanceUpdateByGeneric(ctx=self.ctx)()
+        self.post_instance_update(self.ctx.vars.instance)
+        yield self.TagRulesCreateOrUpdate(ctx=self.ctx)()
```

Comment:
The update changed from `Patch` to `Get+Put`. Is it by design?

Reply:
Yes. 
We don't have Patch call for tag rules. We had to provide update CLI command so it design decision was made previously.